### ISSUE TITLE
Audit #1 – General Overview when logged in – Tooltips aren't fully accessible

### ISSUE THREAD

**Issue by marcus-herrmann** [owncloud/web#4654]
Connected to Audit 1, see #4300 
1.2 General Overview when logged in, URL: https://ocis-a11y.owncloud.works/#/files/list

# Issue
* "Shared with a link" tooltips inside [OcTableFiles](https://owncloud.design/#/oC%20Components/OcTableFiles) are not usable with keyboard only. This is caused because the indicator of a file "shared with a link" (![Screenshot 2021-01-22 at 09 06 16](https://user-images.githubusercontent.com/505181/105463982-2246a300-5c91-11eb-8338-81ccd1e63d60.png)) is not an interactive control, therefore not part of the tabindex. As a consequence, they are not dismissable via keyboard or pointer. WCAG 1.4.13: Content on Hover or Focus and BITV 2.0  Prüfschritt 1.4.13a  demand this https://testen.bitv-test.de/index.php?a=di&iid=111

# Detailed WCAG/BITV requirements:
Source, because very neatly condensed: https://sarahmhigley.com/writing/tooltips-in-wcag-21/

- Hide when a keyboard user presses "Escape" (unless the tooltip will never overlap other content).
- Allow a mouse or pointer user to hide the tooltip, ideally through a close button (unless the tooltip will never overlap other content). The tooltip should not re-appear on subsequent hovers -- think of a zoom user trying to center their view on a specific area without triggering the tooltip and obscuring content.*
- Allow a mouse user to move their mouse over the tooltip content without dismissing the tooltip -- ideally in a manner that does not require laser focus and precision mouse control.
- Remain in view until the user actively dismisses it, or it is no longer valid (e.g. if a loading tooltip appeared, then it could disappear after the content loaded).

# Remediation
Consider a toogletip (https://inclusive-components.design/tooltips-toggletips/) for this exact case instead. For tooltips that belong to an already interactive control, see this CodePen https://codepen.io/smhigley/pen/KjoerX (together with this background article: https://sarahmhigley.com/writing/tooltips-in-wcag-21/)

**Comment by pascalwengerter** [comment]
Closing this since #5018 and #5055 are merged and we're waiting for feedback (and will open new tickets with the requested changes from the `a11y` audit)
